gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/test/java/io/gravitee/rest/api/management/v2/rest/mapper/PageMapperTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright © 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.rest.api.management.v2.rest.mapper;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.gravitee.rest.api.management.v2.rest.model.SourceConfigurationConfiguration;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+/**
+ * @author Sergii ILLICHEVSKYI (sergii.illichevskyi at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+class PageMapperTest extends AbstractMapperTest {
+
+    private PageMapper pageMapper;
+    private ObjectMapper objectMapper;
+
+    @BeforeEach
+    void setUp() {
+        pageMapper = PageMapper.INSTANCE;
+        objectMapper = new ObjectMapper();
+    }
+
+    @Test
+    @DisplayName("Should map SourceConfigurationConfiguration to JSON string")
+    void shouldMapSourceConfigurationConfigurationToJsonString() throws JsonProcessingException {
+        var sourceConfiguration = new SourceConfigurationConfiguration();
+        sourceConfiguration.setAutoFetch(true);
+        sourceConfiguration.setUseSystemProxy(true);
+        sourceConfiguration.setUrl("http://example.com");
+
+        var expectedJsonNode = objectMapper.readTree(objectMapper.writeValueAsString(sourceConfiguration));
+        var actualJsonNode = objectMapper.readTree(pageMapper.map(sourceConfiguration));
+
+        assertEquals(expectedJsonNode, actualJsonNode);
```

Comment:
Instead of comparing the use of the `writeValueAsString` in the mapper vs directly calling the method, WDYT about comparing the expected string? Ex. `"{  \"aFunProperty\": \"value\" }"`. That way we know that the serialization works the way that we want for the database.

Reply:
Done